src/staticdata.c

```diff
@@ -3698,6 +3698,10 @@ JL_DLLEXPORT jl_value_t *jl_restore_package_image_from_file(const char *fname, j
 
     jl_image_t pkgimage = jl_init_processor_pkgimg(pkgimg_handle);
 
+    if (ignore_native){
+        memset(&pkgimage.fptrs, 0, sizeof(pkgimage.fptrs));
+    }
```

Comment:
I am amused that it was this easy. And you checked that we are still getting code-coverage on code that was cached?

Reply:
I'll add a test :)